website/docs/r/wafv2_regex_pattern_set.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+subcategory: "WAFv2"
+layout: "aws"
+page_title: "AWS: aws_wafv2_regex_pattern_set"
+description: |-
+  Provides an AWS WAFv2 Regex Pattern Set resource.
+---
+
+# Resource: aws_wafv2_regex_pattern_set
+
+Provides an AWS WAFv2 Regex Pattern Set Resource
+
+## Example Usage
+
+```hcl
+resource "aws_wafv2_regex_pattern_set" "example" {
+  name        = "example"
+  description = "Example regex pattern set"
+  scope       = "REGIONAL"
+
+  regular_expression {
+    regex_string = "one"
+  }
+
+  regular_expression {
+    regex_string = "two"
+  }
+
+  tags = {
+    Tag1 = "Value1"
+    Tag2 = "Value2"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) A friendly name of the regular expression pattern set.
+* `description` - (Optional) A friendly description of the regular expression pattern set.
+* `scope` - (Required) Specifies whether this is for an AWS CloudFront distribution or for a regional application. Valid values are `CLOUDFRONT` or `REGIONAL`. To work with CloudFront, you must also specify the Region US East (N. Virginia).
```

Comment:
could you also please update the wording here like suggested in #12788 ? 
```
To work with CloudFront, you must also specify the region `us-east-1` (N. Virginia) on the AWS provider.
```

Reply:
@anGie44 done.